packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -25,28 +28,68 @@ const defaultProps = {
   isHovered: false,
   component: 'a',
   isDisabled: false,
-  href: '#'
+  href: '#',
+  keyHandler: () => undefined,
+  index: -1,
+  sendRef: () => undefined
 };
 
-const DropdownItem = ({ className, children, isHovered, component: Component, isDisabled, ...props }) => {
-  const additionalProps = props;
-  if (Component === 'a') {
-    additionalProps['aria-disabled'] = isDisabled;
-    additionalProps.tabIndex = isDisabled ? -1 : additionalProps.tabIndex;
-  } else if (Component === 'button') {
-    additionalProps.disabled = isDisabled;
+class DropdownItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.ref = React.createRef();
```

Comment:
No need for this to be in the constructor. The class properties plugin is being used. This will match other components.

Reply:
there is a babel plugin that pretty much abstracts away any use of the constructor & adds any definition within the class as a class property instead of having to explicitly do so in the constructor.

Reply:
Ah I see, cool. I removed the constructor.